### DISCUSSION TITLE
Different appearance for V3 results web part template configured the same on 2 pages

### DISCUSSION THREAD

**Discussion by webbrewers1** [microsoft-search/pnp-modern-search#2703] [Q&A]
In a v3 results web part, I customized the display template using "manage columns" in the details list option. Wondering why the applied styling works on one page but not another set up exactly the same. It initially worked but then reverted to an unstyled look and I can't fix it. It also works fine in another site so seems to be just this particular page that refuses to comply. When I inspect the code output it's exactly the same.....
Correct look 
![image](https://user-images.githubusercontent.com/54376858/213797212-c3e447c7-186b-435f-9626-3643cd2a117d.png)
Wrong look
![image](https://user-images.githubusercontent.com/54376858/213797461-7d4e89a7-cb8a-4295-8bb0-564a3fbbf286.png)

**Comment by wobba** [comment]
Not sure. I would re-add the web part and config to verify. You can also copy out the web part config from the last web part pane.
![image](https://user-images.githubusercontent.com/51104/213921061-f7fa233d-c08f-453f-8e59-c581b3f37f6f.png)
